package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppeteer-screen-recorder",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
Version updated is not required to be done manually. This repo is configured to used `standard-version` so the version will updated before the release automatically by the build tool. 

Could you please undo the changes.

Reply:
No problem. Done.